client/README.md

```diff
@@ -39,8 +39,8 @@ api_client = client.Client(API_URL, USER_LOGIN, USER_PASSWORD)
 
 # List your registered devices
 devices = api_client.get_my_devices().json()
-# List sites accessible in your scope
-sites = api_client.get_sites().json()
+# List organizations accessible in your scope
+organizations = api_client.get_organizations().json()
```

Comment:
I just realized the README is completely out of date, could you remove the legacy methods?

Reply:
Oh and you created the method "fetch_organizations", let's pick a side and align both

Reply:
Should I create some examples for "camera" (instead of the device bloc) or we can consider that our user should not play with this part of the API, only the engine ?

Reply:
The prime users here are the camera for the detection workflow, the agents through the platform. Illustrating both would be ideal 👍